ramips: correct vendor name for COMFAST/Joowin

araujorm opened the pull request:
When Joowin WR758AC V1 and V2 devices were added, they should have been
added with the primary manufacturer name which is Comfast, since Joowin
is just an alternate vendor name on some coutries or stores.

Correct the vendor name on all files and set Joowin as ALT0 variants.

As a side effect, fix mt76x8 network script which was left incorrectly
unsorted on the case block conditions.

Fixes: 766733e172 ("ramips: add support for Joowin WR758AC V1 and V2")

mans0n commented on target/linux/ramips/image/mt76x8.mk:
```diff
@@ -79,6 +79,30 @@ define Device/buffalo_wcr-1166ds
+  DEVICE_MODEL := WR758AC
+  DEVICE_ALT0_VENDOR := Joowin
+  DEVICE_ALT0_MODEL := WR758AC
+endef
```
Additionally we might set `SUPPORTED_DEVICES` to the former model name for early users.

araujorm replied:
Hello. Is it sufficient to set int in the MakeFile? Or do we also need to add it to the DTS file?

mans0n replied:
In DTS files "compatible" strings are okay now, but the "model" strings should be changed as well.

araujorm replied:
> 

Model strings in the DTS files fixed in another new push I just made.

So it's confirmed that there is no need to add joowin,jw-wr758ac-v{1,2} to the DTS files, I mean, it's enough to add the SUPPORTED_DEVICES entry in the makefile, correct?

mans0n replied:
Precisely, the purpose of `SUPPORTED_DEVICES` is just for sysupgrade without force option.

araujorm replied:
Thanks for the explanation.

araujorm commented:
@mans0n I force pushed the requested changes. Please check my comment above about a doubt I had while adding support for early users, and please confirm if it's good now.

araujorm committed:
ramips: correct vendor name for COMFAST/Joowin

When Joowin WR758AC V1 and V2 devices were added, they should have been
added with the primary manufacturer name which is COMFAST, since Joowin
is just an alternate vendor name on some coutries or stores.

Fix this by changing the the vendor name on the respective files and set
Joowin as ALT0 variants while ensuring compatibility for early users.
Also adjust the model names to better follow the naming rules.

As a side effect, fix mt76x8 network script which was left incorrectly
unsorted on the case block conditions.

Fixes: 766733e172 ("ramips: add support for Joowin WR758AC V1 and V2")
Signed-off-by: Rodrigo Araujo <araujo.rm@gmail.com>

blocktrron commented:
Merged to my staging tree, thanks!
